### PR TITLE
Added first pass of launch blog post content

### PR DESCRIPTION
**Pull request by patcon**
Re-ticketed from https://github.com/hyphacoop/organizing/issues/382

from https://docs.google.com/document/d/1Otvfq0VIJfqevsq5SisB7-uovEDL18CXLwXrzqggl7k/edit#

[screencast](https://recordit.co/iDoCa1ze8D)

### CODE REVIEW

**Comment by patcon** [comment]
We could probably create jekyll `includes` for advanced formatting, if we want anything beyond this. e.g, float right/left caption images, pull quotes, etc.

File: _posts/2021-01-23-how-we-co-operate.md [hunk 1]
```diff
@@ -4,26 +4,170 @@ author: Hypha
...
+(Check out our [playbook][] for the first planning retreat we held in 2018.)
 
-{% highlight ruby %}
-def print_hi(name)
-  puts "Hi, #{name}"
-end
-print_hi('Tom')
-#=> prints 'Hi, Tom' to STDOUT.
-{% endhighlight %}
+   [playbook]: https://github.com/hyphacoop/organizing/blob/master/_posts/2018-december-retreat/2018-11-28-retreat-playbook.md
```
Comment (patcon) [inline _posts/2021-01-23-how-we-co-operate.md]:
```suggestion
   [playbook]: https://link.hypha.coop/retreat-playbook
```

File: _posts/2021-01-23-how-we-co-operate.md [hunk 2]
```diff
@@ -4,26 +4,170 @@ author: Hypha
...
+When we thanked them for their generosity, they’d shrug it off and say, “Principle 6.”
+Seeing the principle of “Co-operation Among Co-operatives” put into practice in everyday interactions was inspiring and reassured us that we were on the right course.
+It’s a phrase we love and have latched onto since.
+
+We’re grateful to [Fourth Pig][], [Anarres][] (RIP),
+[Root Systems][] (RIP), [Fiqus][], [Camba][], [FACTTIC][], [CWCF][],
+and many others who’ve guided us along the way.
+
+   [Fourth Pig]: https://www.fourthpig.org/
+   [Anarres]: http://anarres.ca/
```
Comment (patcon) [inline _posts/2021-01-23-how-we-co-operate.md]:
```suggestion
   [Anarres]: http://web.archive.org/web/20160618102349/https://anarres.ca/
```

File: _posts/2021-01-23-how-we-co-operate.md [hunk 3]
```diff
@@ -4,26 +4,170 @@ author: Hypha
...
+
+We’re excited to hear from you!
+Reach out if there are topics you want to discuss or you think we should cover in future posts.
+Our email is: [dripline@hypha.coop][email]
+
+   [email]: mailto:dripline@hypha.coop
+
+---
+
+This site is published using [Distributed Press][],
```
Comment (patcon) [inline _posts/2021-01-23-how-we-co-operate.md:169]:
```suggestion
This site is published using [Distributed Press][], an initiative we host.
```

File: _posts/2021-01-23-how-we-co-operate.md [hunk 4]
```diff
@@ -4,26 +4,170 @@ author: Hypha
...
+you can [use it to publish][] on the p2p and distributed web too.
```
Comment (patcon) [inline _posts/2021-01-23-how-we-co-operate.md]:
```suggestion
This site is published using [Distributed Press][], an initiative we host.
You can [use it to publish][] on the p2p and distributed web too.
```

**Comment by patcon** [comment]
k, this goes to `staging` branch, so I believe I'm good to merge this so others can take a look at http://staging.hypha.coop before merging into `master`

**Comment by dcwalk** [comment]
1. can we reduce that image of the page to like 55% or so width? It's massive

2. @patcon the topline text that udit drafted for Dripline is nowhere on the blog, can you add it to both the index and the post?
>  A dripline signifies the area under the outermost branches of a tree. It’s this interface between worlds where interesting and productive things happen. 

With a link to https://permacultureprinciples.com/principles/_11/ on "interface"

3. Can we move the distributed press shoutout into the actual page footer? Not sure about this double footer we have going on

4. also I find this text uncessesarily awk:
> This site is published using Distributed Press, an initiative we host. 
can we just say
> This site is published using Distributed Press

No one is going to know what an initiative is and I wonder if that is useful to include? Why not just have less words?

5. Maybe we shout out the initiative in the blog post text like so:

This is the inaugural post of Hypha’s new blog, Dripline **, which we're published on the p2p and distributed web using Distributed Press**. Through Dripline, we’re hoping to share our experiences and reflections as we learn and build technologies together. It’s also an invitation to our friends and neighbours: come help us build the solidarity economy. Principle 6.

**Comment by benhylau** [comment]
1. It's important that this goes into staging first https://github.com/hyphacoop/hypha.coop/pull/22 along with my comments, pls ping @YurkoWasHere to verify
2. https://ipfs.distributed.press/ipns/hypha.coop/ is broken bc the asset files are still not properly referenced (see [my comment here](https://github.com/hyphacoop/hypha.coop/pull/17#issue-560485907))
3. Can we pls put a solid background around the text this is offensive to readers:

    Screen Shot 2021-02-15 at 4 10 42 PM

**Pull request merged**